go/tools/semconvgen/README.md

```diff
@@ -0,0 +1,23 @@
+# Go Semantic Convention Generator
+
+This tool is designed to generate constants in a semantic convention package for the Go API and the collector.
+It may be used by other systems, but it's primary function beyond invoking the template processor is to ensure that
+generated identifiers conform to Go's naming idiom, particularly with respect to initialisms and acronyms.
```

Comment:
Please mention here that this primary function could also be fulfilled just with the Python tool.

Reply:
As mentioned in another comment, happy to review a PR that helps us with that change :)